src/routes/transactionWrapper.js

```diff
@@ -1,9 +1,22 @@
 import { sequelize } from '../models';
+import handleErrors from '../lib/apiErrorHandler';
+
+const namespace = 'SERVICE:WRAPPER';
+
+const logContext = {
+  namespace,
+};
 
 export default function transactionWrapper(originalFunction) {
   return async function wrapper(req, res, next) {
-    return sequelize.transaction(async () => {
-      const result = await originalFunction(req, res, next);
+    return sequelize.transaction(async (t) => {
+      let result;
+      try {
+        result = await originalFunction(req, res, next);
+      } catch (err) {
+        t.rollback();
```

Comment:
```suggestion
        await t.rollback();
```

Reply:
looks like as an alternative you can just throw an error 
https://sequelize.org/docs/v6/other-topics/transactions/#throw-errors-to-rollback